ls1intum/Artemis
# SAML2: Fix disabled accounts

### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the KIT test server.

### Motivation and Context
A login via SAML2 did not respect the activation status of users.

### Description
The server checks if a user was disabled. If this is the case a 403 Forbidden is returned. The client shows the message via jHispster Alerts

### Steps for Testing
1. Disable a SAML2 user
2. Login with ether SAML2 or password both should deny the login.

### Side note
For further authentication JWT-Tokens are used in SAML2, facing the issue same issue #3139. 

@dfuchss
